src/main/java/frc/robot/subsystems/swerve/Swerve.java

```diff
@@ -215,12 +215,8 @@ public void setModulesState(SwerveModuleState[] moduleStates, boolean preventJit
         }
     }
 
-    private double getYawDegreesCW() {
-        return gyroYawHelper.getAngleDegreesCW();
-    }
-
-    private double getYawDegreesCCW() {
-        return gyroYawHelper.getAngleDegreesCCW();
+    private Rotation2d getYawCCW() {
+        return gyroYawHelperCCW.getAngle();
```

Comment:
Isn't getRotation2d better?

Reply:
I don't think so. Rotation2d is a type, it doesn't tell you what the value of it means.